README.md

```diff
@@ -298,7 +299,7 @@ You can also retrieve the same info for the user that is currently logged-in.
 _Get information about the currently logged-in user_
 
 ```csharp
-var user = await e621Client.GetUserAsync("noppes");
+var user = await e621Client.GetLoggedInUserAsync("noppes");
```

Comment:
Nice catch! Sadly, it's still not correct. This should be `var user = await e621Client.GetLoggedInUserAsync();` ;3